### ISSUE TITLE
[RC4] FBSystemService shutdownWithOptions is restarting instead of shutting down

### ISSUE THREAD

**Issue by ichitaso** [pwn20wndstuff/Undecimus#11]
Even if you try to shut down, it becomes Reboot.  

- When turning off the power manually
- The following command
  
```
pid_t pid;
int status;
const char* args[] = {"/sbin/halt", NULL};
posix_spawn(&pid, args[0], NULL, NULL, (char* const*)args, NULL);
waitpid(pid, &status, WEXITED);
```
  
`[[%c(FBSystemService) sharedInstance] shutdownWithOptions:0];`

**Comment by Cryptiiiic** [comment]
@ichitaso Just to reclarify, this issue resides with someone else other than pwn, trying to get him to fix soon.

**Comment by ichitaso** [comment]
Fixed this issue on unc0ver 2.1.0, thanks!
